rpcs3/Emu/RSX/overlays.h

```diff
@@ -353,6 +360,19 @@ namespace rsx
 					m_list->add_entry(new_stub);
 				}
 
+				if (!m_list->m_items.size())
+				{
+					m_no_saves_text = std::make_unique<label>();
+					m_no_saves_text->set_font("Arial", 20);
+					m_no_saves_text->set_pos(515, 345);
```

Comment:
Set the pos and size to span the whole width of the list view and set text alignment to center. Remember not all fonts have the same metrics.